fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/method/message/data/election/ElectionEncryptedVote.java

```diff
@@ -24,8 +24,8 @@ public class ElectionEncryptedVote {
    * @param questionId id of the question
    * @param encryptedVote encrypted uniaue indice of the chosen vote
    * @param writeInEnabled indicates if write in is enabled
-   * @param encryptedWriteIn
-   * @param electionId
+   * @param encryptedWriteIn whether the election allows write-in
```

Comment:
It would not be that big of change tough it would be a couple lines across multiple files (those involved with UI too). Therefore since it is also not the purpose of this PR, we could do a small separate PR with that.